### ISSUE TITLE
About training loss problems

### ISSUE THREAD

**Issue by zardkk** [Shuaitaoaau/SPP#1]
Hello, can you share the Model_Fit   training code? I meet a question that the mse loss couldn't descend well,  and mse loss stops decreasing once it reaches 0.16.
